### DISCUSSION TITLE
Possible name for project

### DISCUSSION THREAD

**Discussion by ninkibah** [pkeir/cest#16] [General]
Paul,
I was not quite right about the name of the Italian wine I mentioned on Saturday. I's called "est, est, est". In Italian, as in Latin, pronouns aren't necessary. Anyway, sadly, an alternative origin story for the name of the project won't work :-(

**Comment by pkeir** [comment]
Hi Jonathan,

Thanks. After the talk I did search, and wonder; as I had also heard the story, but couldn't remember "est, est, est".

Anyway, I'm glad now to get a second chance to pass on the (somewhat meagre) origin story of the name; it had slipped my mind last week. The inspiration comes from the phrase "C'est la vie". On its own, the "C'est" part of that is somewhat hollow (perhaps "it is", "this is", or "that is"), but I still like it. It's also only 4 letters, which is tolerable for a C++ namespace name; and the "st" therein has just a whiff of "std" :-)

Cheers,
